RandomData.Api/StringGeneration/StringGenerationServiceHelpers.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using RandomData.Api.StringGeneration.ServiceImplementations;
+using RandomData.Api.StringGeneration.Validators;
+
+namespace RandomData.Api.StringGeneration
+{
+    public static class StringGenerationServiceHelpers
```

Comment:
By conventions name IServiceCollection extensions are created by adding postfix Extenssions to name of "module". StringGenerationExtensions is ok.